components/script/dom/intersectionobserver.rs

```diff
@@ -284,10 +365,19 @@ impl IntersectionObserverMethods<crate::DomTypeHolder> for IntersectionObserver
     }
 }
 
+impl Clone for ElementOrDocument {
```

Comment:
Let's derive this instead: https://github.com/servo/servo/blob/main/components/script_bindings/codegen/Bindings.conf#L607